src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StorageDaoImplTest {
+    private static StorageDao storageDao;
+    private static RegistrationService registrationService;
+    User user1;
+    User user2;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        Storage.people.clear();
+        user1 = new User();
+        user2 = new User();
```

Comment:
May be better to Beforeall?

Reply:
ok. i've done that. but in this case I had to make users static since beforeall is a static method